docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc

```diff
@@ -978,6 +978,81 @@ If you want to use the server PacketFence to authenticate users on the Brocade s
 
 NOTE: Make sure to have a local account in case the switch can not reach the PacketFence server
 
+ICX 6600 Series
+^^^^^^^^^^^^^^^
```

Comment:
Could you use new AsciiDoc syntax for [titles](https://packetfence.org/doc/PacketFence_Developers_Guide.html#_titles) ?

Reply:
Sure, I have no clue about the conversion, I'm just using the same title level as the other. ^^^ translate to what?

Reply:
See my link in my previous comment.